Week_2/functions/reverse/reverse.cpp

```diff
@@ -0,0 +1,24 @@
+#include <string>
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+void Reverse(vector<int> &source)
+{
+    reverse(source.begin(), source.end());
+}
```

Comment:
Redundand. You use func once and created it only to hide begin/end. Not a c++ style.

Reply:
but it was my task, no?

Reply:
Didn't see the task because you pushed only the code.